library-no-op/build.gradle

```diff
@@ -11,6 +11,10 @@ android {
         ]
     }
 
+    sourceSets.all {
+        kotlin.srcDir("src/$name/kotlin")
+    }
```

Comment:
Is it necessary? If I recall correctly they recently changed AGP to automatically include `kotlin` source sets.

Reply:
I also remember that AGP should handle that, but didn't check. Was keeping in mind that back in spring had to add such construction.

Reply:
Will check if it works without and update PR, thanks.

Reply:
Yeah, looks like everything works without adding `sourceSets` lines in `build.gradle` files. Updated.